xoai-common/src/main/java/io/gdcc/xoai/xml/CopyElement.java

```diff
@@ -109,7 +108,7 @@ protected void writeXml(XmlWriter writer) throws IOException {
 
             String firstChars = new String(bytes, StandardCharsets.UTF_8);
             // match the start with the compiled regex and replace with nothing when matching.
-            firstChars = xmlDeclaration.reset(firstChars).replaceFirst("");
+            firstChars = xmlDeclaration.matcher(firstChars).replaceFirst("");
 
             // write the chars to the output stream
             writer.getOutputStream().write(firstChars.getBytes(StandardCharsets.UTF_8));
```

Comment:
I'm just leaving this comment at the bottom.

The Sonar test is failing with the same error we saw in this issue:

- #273